x-pack/plugins/serverless_search/tsconfig.json

```diff
@@ -9,11 +9,9 @@
     "public/**/*.ts",
     "public/**/*.tsx",
     "server/**/*.ts",
-    "../../../typings/**/*",
```

Comment:
why did you change this part of the config?

Reply:
Updated.